docs/install_guides/installing-model-dev-tools.md

```diff
@@ -73,96 +89,36 @@ Activate the newly created Python virtual environment by issuing this command:
-
-   For example, to install and configure the components for working with TensorFlow 2.x and ONNX, use the following command:
-
-   .. code-block:: sh
-
-      pip install openvino-dev[tensorflow2,onnx]
-
-   .. note::
-
-      Model Optimizer support for TensorFlow 1.x environment has been deprecated. Use the `tensorflow2` parameter to install a TensorFlow 2.x environment that can convert both TensorFlow 1.x and 2.x models. If your model isn't compatible with the TensorFlow 2.x environment, use the `tensorflow` parameter to install the TensorFlow 1.x environment. The TF 1.x environment is provided only for legacy compatibility reasons.
-
-.. tab:: C++
-
-   When using OpenVINO Development Tools for C++ development, it’s important to install the same version as OpenVINO Runtime. Following the instructions below will ensure that you are installing a version that matches that of OpenVINO Runtime.
-
-   **Recommended: Install Using the Requirements Files**
-
-   After you have installed OpenVINO Runtime from an archive file, you can find a set of requirements files in the <INSTALL_DIR>\tools\ directory. The requirements files will install the matching version of OpenVINO Development Tools and its dependencies.
-
-   1. Install the OpenVINO Development Tools mandatory requirements using the following command:
-
-      .. code-block:: sh
-
-         pip install -r <INSTALL_DIR>\tools\requirements.txt
-
-   2. If you are using additional frameworks, you must also install the requirements for those frameworks using the corresponding requirements file. For example, if you are using a TensorFlow model, use the following command to install requirements for TensorFlow:
-
-      .. code-block:: sh
-
-         pip install -r <INSTALL_DIR>\tools\requirements_tensorflow2.txt
-
-   **Alternative: Install the openvino-dev Package from PyPI**
-
-   You can also install OpenVINO Development Tools from PyPI using the following command.
-
-   .. important::
-
-      Make sure that the `openvino-dev` version you specified matches your installed version of OpenVINO Runtime. Otherwise, compatibility errors are likely to occur.
-
-   .. code-block:: sh
-
-      pip install openvino-dev[EXTRAS]==2022.3.0
-
-   where the EXTRAS parameter specifies one or more deep learning frameworks via these values: caffe, kaldi, mxnet, onnx, pytorch, tensorflow, tensorflow2. Make sure that you install the corresponding frameworks for your models. For example: 
-
-   .. code-block:: sh
-
-      pip install openvino-dev[tensorflow2,onnx]==2022.3.0
+```sh
+pip install openvino-dev[extras]
+```
 
-   .. note::
+where the `extras` parameter specifies the source deep learning framework for the input model
+and has the following values:  `caffe`, `kaldi`, `mxnet`, `onnx`, `pytorch`, `tensorflow`, `tensorflow2`. 
+Make sure that you install the corresponding frameworks for your models.
```

Comment:
Not sure if "make sure .." comment actually adds value here, not sure what you think...

Reply:
Removed.